beater/api/config/agent/handler.go

```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/elastic/beats/libbeat/monitoring"
```

Comment:
group imports

Reply:
Not sure why but my `goimports -local` settings had been removed again ..